src/libs/OptionsListUtils.js

```diff
@@ -213,20 +213,22 @@ function getParticipantsOptions(participants, personalDetails) {
     const details = getPersonalDetailsForAccountIDs(_.pluck(participants, 'accountID'), personalDetails);
     return _.map(participants, (participant) => {
         const detail = details[participant.accountID];
+        const login = detail.login || participant.login;
+        const displayName = detail.displayName || LocalePhoneNumber.formatPhoneNumber(login);
```

Comment:
I don't think we need a fallback here either. `getPersonalDetailsForAccountIDs` will return the data from onyx and `displayName` seems always set.

Reply:
The fallback is for a case where the user doesn't exist, for example when you search a random email. More explanation in my previous comment here https://github.com/Expensify/App/pull/17964#issuecomment-1600085935